### DISCUSSION TITLE
Run Collection without Using Stored Cookies

### DISCUSSION THREAD

**Discussion by orimoloyeb** [thunderclient/thunder-client-support#1073] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/79852158/226927877-1243024b-e3f0-4b0c-8a57-b9489071ce8f.png)

Postman has the option to run a collection without using stored cookies. This allows such that if there are two requests A and B in the collection, and the collection is run at once, request B runs without using any of the cookies created by request A. 

Is this possible in Thunder Client? @rangav

**Comment by rangav** [comment, marked as answer]
Hi @orimoloyeb Currently not possible, please create a feature request, Will add to roadmap.
